autoload/go/term.vim

```diff
@@ -60,9 +56,9 @@ function! go#term#newmode(bang, cmd, mode) abort
 
   " we are careful how to resize. for example it's vertical we don't change
   " the height. The below command resizes the buffer
-  if a:mode == "split"
+  if mode =~ "split"
     exe 'resize ' . height
-  elseif a:mode == "vertical"
+  elseif mode =~ "vertical"
```

Comment:
matching on a regular expression here would change something like `left vertical` to `vertical resize`. Is that what you intended?

Reply:
No, I'm not quite sure how it works when set mode as `vertical`. In fact, when I set `let g:go_term_mode = "vertical"`, following error occured:

![image](https://user-images.githubusercontent.com/2898670/33949495-bbe1c1cc-e064-11e7-8f73-0ef548bf1d72.png)

It should be something like `vsplit`?

Thanks!